vars/runIntegTestScript.groovy

```diff
@@ -55,13 +55,16 @@ void call(Map args = [:]) {
     String switchCommandStart = switchUser.equals('true') ? "su `id -un 1000` -c \"env PATH=\$PATH $javaHomeCommand" : "env PATH=\$PATH $javaHomeCommand"
     String switchCommandEnd = switchUser.equals('true') ? '"' : ''
 
+    String testManifest = "manifests/${args.testManifest}"
```

Comment:
This would be a breaking change for the lib right? Looking at https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch-dashboards/integ-test.jenkinsfile#L286?
Maybe bump the major version.

Reply:
Don't think it will be breaking, we are just moving the path details from jenkinsfile to script. 
I am okay to do a major version bump.

Reply:
Done